include/libdnf/base/goal.hpp

```diff
@@ -54,53 +54,285 @@ class Goal {
     explicit Goal(const libdnf::BaseWeakPtr & base);
     explicit Goal(libdnf::Base & base);
     ~Goal();
+
+    // TODO(jmracek) Implemented not yet
     void add_module_enable(const std::string & spec);
-    /// Prevent reinstallation by adding of already installed packages with the same NEVRA
+
+    /// Add install job to the goal for future resolvemnt. The operation will not result in package reinstall when a
```

Comment:
typo - resolvemnt -> resolvem**e**nt. (repeated in many places)

Reply:
I'd also prefer to use `dependency resolution` rather than `resolvement`

Reply:
In this case it is not a `dependency resolution`, but transformation of `spec` into Ids.